assets/stylesheets/result_preview.scss

```diff
@@ -91,7 +91,6 @@
     // live view
     canvas {
         max-width: 100%;
-        border: #ddd 1px solid;
```

Comment:
I think the border can give some good indication what's going on if there's a problem so consider to keep that

Reply:
yes, though it is then visible during loading
![canvas_border](https://github.com/user-attachments/assets/ea07412e-a9b9-4158-8646-df92ec265710)
i tried including the loading animation within the canvas, though that hid it.

Reply:
> i tried including the loading animation within the canvas, though that hid it.

Maybe it helps to assign a high z-index to the loading animation.